doc/api/stream.md

```diff
@@ -1895,10 +1893,10 @@ const fs = require('fs');
 async function run() {
   await pipeline(
     fs.createReadStream('lowercase.txt'),
-    async function* (source) {
+    async function* (source, signal) {
       source.setEncoding('utf8');  // Work with strings rather than `Buffer`s.
       for await (const chunk of source) {
-        yield chunk.toUpperCase();
+        yield await processChunk(chunk, { signal });
```

Comment:
Unless `processChunk()` here handles it, the `signal` should likely be checked at each iteration here.

Reply:
Process chunk should handle it. Checking it in the loop should not be necessary as the consumer will return.